src/Command/Project/ProjectSshConfigCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Platformsh\Cli\Command\Project;
+
+use Platformsh\Cli\Command\CommandBase;
+use Platformsh\Cli\Console\AdaptiveTableCell;
+use Platformsh\Cli\Util\Table;
+use Platformsh\Client\Model\Project;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ProjectSshConfigCommand extends CommandBase
+{
+
+    protected function configure()
+    {
+        $this
+            ->setName('project:ssh-config')
+            ->addOption('alias', null, InputOption::VALUE_OPTIONAL, 'Specify a custom alias the project')
+            ->setAliases(['ssh-config'])
+            ->setDescription('outputs OpenSSH valid configuration to connect all of the project environments');
+        $this->addProjectOption();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->validateInput($input);
+        $project = $this->getSelectedProject();
+
+        $environments = $this->api()->getEnvironments($this->getSelectedProject(), $refresh ? true : null);
```

Comment:
The $refresh variable doesn't exist, and you've already got a $project variable, so you probably want:

```php
$environments = $this->api()->getEnvironments($project);
```